standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/MetaStoreDirectSql.java

```diff
@@ -2960,4 +2977,22 @@ public void lockDbTable(String tableName) throws MetaException {
       throw new MetaException("Error while locking table " + tableName + ": " + sqle.getMessage());
     }
   }
+
+  public void deleteColumnStatsState(long tbl_id) throws MetaException {
+    // @formatter:off
+    String queryText = ""
+        + "delete from " + PARTITION_PARAMS + " pp"
+            + " using " + PARTITIONS + " p "
+            + " where "
+            + "   p.\"PART_ID = pp.\"PART_ID\""
```

Comment:
Please carefully check every ". Postgres and MySql are very problematic

Reply:
I've changed the sql - now it works with mysql as well